docs/dev/dev-mode.md

```diff
@@ -11,72 +11,103 @@ Description: Developer mode (dev mode) is intended to be used for testing and de
 
 # Developer Mode Installation
 
-Developer mode (dev mode) is intended to be used for testing and development purposes.
-
-This video shows the dev mode installation.
-
-<iframe width="950" height="475" src="https://www.youtube.com/embed/TG0GaAD_6J4" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+!!! attention
+    Developer mode (dev mode) is intended to be used for local testing and development purposes.
 
 ## Requirements
 
+- The Kubernetes node must pass the [host-check](https://raw.githubusercontent.com/harvester/harvester/master/hack/host-check.sh)
+- If the Kubelet's RootDir is not `/var/lib/kubelet`, you must create a `bind mount` to `/var/lib/kubelet` as follows:
+   ```bash
+   KUBELET_ROOT_DIR="path to your kubelet root dir"
+   echo "${KUBELET_ROOT_DIR} /var/lib/kubelet none bind 0 0" >> /etc/fstab
+   mkdir -p /var/lib/kubelet && mount -a
 - [**Multus**](https://kubernetes.io/docs/concepts/cluster-administration/networking/#multus-a-multi-network-plugin) is installed across your cluster and a corresponding `NetworkAttachmentDefinition` CRD is created.
```

Comment:
Do we require Longhorn and KubeVirt to be installed as well? Or it's a part of the chart.

Reply:
Harvester chart contains Longhorn and KubeVirt

Reply:
Can we call that out here as well?

Reply:
Added